plugins/woocommerce/tests/performance/requests/api/orders.js

```diff
@@ -0,0 +1,212 @@
+/* eslint-disable no-shadow */
+/* eslint-disable import/no-unresolved */
+/**
+ * External dependencies
+ */
+import encoding from 'k6/encoding';
+import http from 'k6/http';
+import { check, group } from 'k6';
+import { findBetween } from 'https://jslib.k6.io/k6-utils/1.2.0/index.js';
+
+/**
+ * Internal dependencies
+ */
+import {
+	base_url,
+	admin_username,
+	admin_password,
+	addresses_guest_billing_first_name,
+	addresses_guest_billing_last_name,
+	addresses_guest_billing_country,
+	addresses_guest_billing_address_1,
+	addresses_guest_billing_address_2,
+	addresses_guest_billing_city,
+	addresses_guest_billing_state,
+	addresses_guest_billing_postcode,
+	addresses_guest_billing_phone,
+	addresses_guest_billing_email,
+} from '../../config.js';
+
+export function ordersAPI() {
+	const credentials = `${ admin_username }:${ admin_password }`;
+	const encodedCredentials = encoding.b64encode( credentials );
+	const requestHeaders = {
+		Authorization: `Basic ${ encodedCredentials }`,
+		'Content-Type': 'application/json',
+	};
+
+	const createData = {
+		billing: {
+			first_name: `${ addresses_guest_billing_first_name }`,
+			last_name: `${ addresses_guest_billing_last_name }`,
+			address_1: `${ addresses_guest_billing_address_1 }`,
+			address_2: `${ addresses_guest_billing_address_2 }`,
+			city: `${ addresses_guest_billing_city }`,
+			state: `${ addresses_guest_billing_state }`,
+			postcode: `${ addresses_guest_billing_postcode }`,
+			country: `${ addresses_guest_billing_country }`,
+			email: `${ addresses_guest_billing_email }`,
+			phone: `${ addresses_guest_billing_phone }`,
+		},
+		shipping: {
+			first_name: `${ addresses_guest_billing_first_name }`,
+			last_name: `${ addresses_guest_billing_last_name }`,
+			address_1: `${ addresses_guest_billing_address_1 }`,
+			address_2: `${ addresses_guest_billing_address_2 }`,
+			city: `${ addresses_guest_billing_city }`,
+			state: `${ addresses_guest_billing_state }`,
+			postcode: `${ addresses_guest_billing_postcode }`,
+			country: `${ addresses_guest_billing_country }`,
```

Comment:
Do we need to use string literals here? We're already exporting strings from the config, isn't it?

Reply:
I think they are needed to substitute in the parameter

Reply:
Ahhh perhaps this is to cast every time (such as integers, etc) to strings in the object?